dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -364,63 +369,147 @@ protected ModifyJobPriorityModel call() throws Exception {
+                    if (closedSession != null) {
+                        CancelJobsWorker.cancelAllRunningJobsBySession(jobWorkers, jobInterruptionStore, LOG, workers, closedSession);
+                        store.removeSession(closedSession);
+                        ds3Common.getExpandedNodesInfo().remove(closedSession.getSessionName() +
+                                StringConstants.SESSION_SEPARATOR + closedSession.getEndpoint());
+                        ds3Common.setSessionOfClosedTab(null);
+                        ds3PathIndicator.setText(StringConstants.EMPTY_STRING);
+                        ds3PathIndicatorTooltip.setText(null);
+                        deepStorageBrowserPresenter.logText(closedSession.getSessionName() +
+                                StringConstants.SESSION_SEPARATOR + closedSession.getEndpoint() + StringConstants
+                                .SPACE + resourceBundle.getString("closed"), LogType.ERROR);
+                    }
+                }
+                final Session currentSession = getSession();
+                if (currentSession != null) {
+                    final Map<String, FilesAndFolderMap> jobIDMap = ParseJobInterruptionMap.getJobIDMap(jobInterruptionStore.getJobIdsModel().getEndpoints(),
+                            currentSession.getEndpoint() + StringConstants.COLON + currentSession.getPortNo(),
+                            deepStorageBrowserPresenter.getJobProgressView(), null);
+                    ParseJobInterruptionMap.setButtonAndCountNumber(jobIDMap, deepStorageBrowserPresenter);
+                }
+            } catch (final Exception e) {
+                LOG.error("Failed to remove session", e);
+            }
+            ds3PathIndicator.setText(StringConstants.EMPTY_STRING);
+            ds3PathIndicatorTooltip.setText(StringConstants.EMPTY_STRING);
+            if (store.size() == 0) {
+                addNewTab.setTooltip(null);
+            }
+        }
     }
 
+    private void setBlank(final boolean isSetBlank) {
+        if (isSetBlank) {
+            ds3PathIndicator.setText(StringConstants.EMPTY_STRING);
+            ds3PathIndicatorTooltip.setText(StringConstants.EMPTY_STRING);
+            capacityLabel.setVisible(false);
+            infoLabel.setVisible(false);
+        } else {
+            capacityLabel.setVisible(true);
+            infoLabel.setVisible(true);
+            capacityLabel.setText(resourceBundle.getString("infoLabel"));
+            infoLabel.setText(resourceBundle.getString("infoLabel"));
+        }
+    }
+
+    private void ds3NewBucket() {
+        LOG.info("Create Bucket Prompt");
+        final Session session = getSession();
+        if (session != null) {
+            deepStorageBrowserPresenter.logText(resourceBundle.getString("fetchingDataPolicies"), LogType.INFO);
+            final Task<CreateBucketWithDataPoliciesModel> getDataPolicies = new Task<CreateBucketWithDataPoliciesModel>() {
```

Comment:
The code is already modified to get the DataPolicies as follows:

Using Ds3Client.getDataPoliciesSpectraS3()